bin/filtered_npm_audit.sh

```diff
@@ -0,0 +1,307 @@
+#!/bin/bash
```

Comment:
IIUC, in order for this to run on all platforms, it should get the path to `bash` from `/usr/bin/env bash`.

But also, at first glance, it looks like the only real bashisms are the locals, so could it also be written POSIX-compliant for `/bin/sh`?